### DISCUSSION TITLE
ODOO Update versions V.15 to V.17 - Historical data.

### DISCUSSION THREAD

**Discussion by hferrer0610** [OCA/odoo-community.org#172] [Q&A]
Hello everyone.

I am doing an upgrade project from version 15 to 17, I am not going to migrate historical data, I will leave the historical data in version 15, up to the date of Go Live, and, in version 17 I will start with initial balances and operations from scratch , example: New Purchase Orders, New customer orders, etc. 

My questions are:
1. Is there a way to extract information from data that I will leave in version 15, for example: historical sales data, processed purchase orders, historical inventory data, etc.?
2.  Is there a connector or application for I can use Power BI to manage this data and be able to do reports, demand analysis, inventory analysis, etc.? 

Thank you all in advance for your comments and recommendations. 

Regards.

**Comment by vdewulf** [comment]
For the sake of consistency, this question was also asked on the Contributors mailing list and got several answers, visible here:
https://odoo-community.org/groups/contributors-15/contributors-1939946?mode=thread&date_begin=&date_end=
